packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.md

```diff
@@ -64,13 +65,15 @@ const styles = StyleSheet.create({
 </View>;
 ```
 
-### Example: Custom OnePaneDialog with long content (overflows)
+### Example: Custom Footer (3 actions)
```

Comment:
This example is awesome.  A comment here to encourage people to resize the page so that they can see what the mobile version looks like would be helpful.  One minor issue is the tertiary buttons look weird at 100%. Can you file a bug for that?
<img width="413" alt="Screen Shot 2019-05-09 at 7 09 43 PM" src="https://user-images.githubusercontent.com/1044413/57492233-15e89f80-728e-11e9-8d40-8ecf7f9d451d.png">

Reply:
Nice catch, I'll create that ticket :)